draft-ietf-dnsop-serve-stale.md

```diff
@@ -297,25 +299,24 @@ document.
 
 ## Implementation considerations
 
-This document mainly describes the method / concept behind serving stale data, and intentionally does not provide a formal algorithm. The concept is not overly complex, and we believe that it would be hubris to believe that we know better than resolver authors how exactly to implement the concept in their code-base. The processing of serve-stale is a local operation, and consitent variables between implementations (or instances) is not needed for interoperability. 
+This document mainly describes the method / concept behind serving stale data, and intentionally does not provide a formal algorithm. The concept is not overly complex, and the details are best left to resolver authors to implement the concept in their code-base. The processing of serve-stale is a local operation, and consitent variables between implementations (or instances) are not needed for interoperability.
 
```

Comment:
... fine. I liked my hubris comment, but this is probably more readable... <grump>